packages/multichain-account-service/src/MultichainAccountService.ts

```diff
@@ -105,14 +106,14 @@ export class MultichainAccountService {
     this.#providers = [
       new EvmAccountProvider(
         this.#messenger,
-        providerConfigs?.[EvmAccountProvider.NAME],
+        providerConfigs?.[EVM_ACCOUNT_PROVIDER_NAME],
```

Comment:
Same here:

```console
packages/multichain-account-service/src/MultichainAccountService.ts:109:9 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ EVM?: EvmAccountProviderConfig | undefined; Solana?: SnapAccountProviderConfig | undefined; }'.
  No index signature with a parameter of type 'string' was found on type '{ EVM?: EvmAccountProviderConfig | undefined; Solana?: SnapAccountProviderConfig | undefined; }'.
```